cluster/src/main/scala/org/apache/spark/sql/streaming/SnappyStreamingContextFactory.scala

```diff
@@ -61,13 +56,18 @@ class SnappyStreamingContextFactory extends SparkContextFactory {
     new SnappyStreamingContext(LeadImpl.getInitializingSparkContext,
       Milliseconds(interval)) with ContextLike {
 
+      private val addedJars = scala.collection.mutable.ArrayBuffer.empty[String]
```

Comment:
But addJobJar can be called by multiple threads no? Also didn't see any caller for addJobJar.